### ISSUE TITLE
new emoji panle on windos 11 isn't accessible for nvda.

### ISSUE THREAD

**Issue by alireza1385mamani** [nvaccess/nvda#13104]
new emoji panle on windos 11 isn't accessible with screen reader.
### Steps to reproduce:
1- load windos or any other app like notepad that you want to write some emojies.
2- press windos key +full stop .
3- press any key to navigate throo emojis.
### Actual behavior:
it is saying blank and you can not press enter on a emoji to write.
### Expected behavior:
it has to navigate throo emojis and read the emojis and when you press the enter key it has to write the emojis on the notepad or any other word processer that you are in.
### System configuration
 NVDA installed
instaled the beta version of nvda.
#### NVDA version:
nvda 2021.3 beta 2.
#### Windows version:
Windows 11 21H2 (x64) build 22000.348
#### Does the issue still occur after restarting your computer?
yes it does.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes. it happens on every version of nvda.
#### If NVDA add-ons are disabled, is your problem still occurring?
yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes.
[nvda.log](https://github.com/nvaccess/nvda/files/7618575/nvda.log)

**Comment by alireza1385mamani** [comment]
if you need anything, please let me know.
thanks

**Comment by fernando-jose-silva** [comment]
according to my tests, if the add-on
NVDAExtensionGlobalPlugin-
is installed nvda continues to speak in white.
which is a shame, I really like this add-on.


Another question, if the option to
Auto focus mode when focus changes
is unchecked, when walking with the arrow below in the list of emogis, nvda is announcing windows input experience for several times, making the use of this window unfeasible.
For this second point, if a solution were possible, it would be very interesting.

**Comment by josephsl** [comment]
Hi, just to make sure: what happens if you enable Windows App Essentials add-on? If that solves the problem, then it gives me a good reason to write a pull request this week. Thanks.

**Comment by fernando-jose-silva** [comment]
if I enable the add-on
Windows App Essentials;
everything works though:
even with the complement
Windows App Essentials;
installed, if the add-on
NVDAExtensionGlobalPlugin-
is installed will not work, the blank ad continues.
it is also necessary that the option
Auto focus mode when focus changes
is checked.
if it is unchecked, an ad of windows input experience is constantly informed when walking with the arrow below, making the use of the window unfeasible.
in short, currently for everything to work it is necessary
install the addon
Windows App Essentials;
uninstall the addon
NVDAExtensionGlobalPlugin-
check the option
Auto focus mode when focus changes

**Comment by CyrilleB79** [comment]
Cc @paulber19 for information regarding the "NVDAExtensionGlobalPlugin:" add-on. Maybe a patched function that is not up-to-date in the add-on.

**Comment by josephsl** [comment]
Hi,

The resulting PR is trivial and not so trivial:

* Trivial: only involes overlay class demotion (reomving editable text scripts)
* Nont so trivial: Im sure there is a much better strategy than removing an overlay class

Rest assured that a resolution is in the works.

**Comment by alireza1385mamani** [comment]
thanks for youre good sujestion. it really worked.
even the problem of switching languages is fixed.

**Comment by alireza1385mamani** [comment]
This issue hasen't been fixed yet.
we have to use windos app esencials addon to fix that. and now I'm using the rc 2021.3 of nvda.

**Comment by josephsl** [comment]
Hi, a fix was submitted for inclusion in the next version of NVDA (2022.1) as it was too late for 2021.3. Thanks.
